source/_integrations/daikin.markdown

```diff
@@ -112,3 +112,16 @@ Zones with the name `-` will be ignored, just as the AirBase application is work
 </div>
 
 Additionally the Daikin Streamer (air purifier) function can be toggled on supported devices using a switch.
+
+## Region Changing
+
+The European and United States controllers (Most likely the Australian controllers too) have an http api endpoint that allows you to change the controllers region so that other regional apps can be used. (Sometimes these controllers get exported to regions that can not download the app for the controllers region)
```

Comment:
```suggestion
The European and United States controllers (Most likely the Australian controllers too) have an HTTP API endpoint that allows you to change the controllers region so that other regional apps can be used. (Sometimes these controllers get exported to regions that can not download the app for the controllers region.)
```